portfolio/index.md

```diff
@@ -7,40 +7,40 @@ nav:
 
 # Portfolio
 
-Our past, present, and future projects with our various clients.
-{:.center}
+{% include section.html %}
 
-{% include link.html link="https://greenelab.com/" text="Greene Lab" icon="fas fa-flask" %}
-{% include link.html link="https://tislab.org/" text="TIS Lab" icon="fas fa-flask" %}
-{% include link.html link="https://medschool.cuanschutz.edu/ai" text="The Center itself" icon="fas fa-flask" %}
-{:.center}
+## Collaborators
 
-**Filter by type:**
+Some of the individual groups we work with:
 {:.center}
-{%
-  include tags.html
-  tags="website, frontend, backend, devops, UI"
+
+{% 
+  include gallery.html
+  link1="https://greenelab.com/"
```

Comment:
Is there a way to get these links to open in a new window/tab, e.g. by setting `target="_blank"` on the links? 

UPDATE: I looked into it a bit and it's possible to change Jekyll's renderer from GitHub-flavored markdown to a different style, "Kramdown", that allows you to add arbitrary attributes to tags, but supposedly it can change the site's rendering in subtle ways. Maybe it's best to just leave the links as-is to avoid changing the renderer, but I figured I'd leave a record of it all the same. (reference, https://stackoverflow.com/a/71373125/346905)

Reply:
I created this issue:

https://github.com/greenelab/lab-website-template/issues/130